README.md

```diff
@@ -448,6 +448,11 @@ As soon as that PR is merged and a new version released, all you'll have to do i
 
 The [vscode-styled-components](https://github.com/styled-components/vscode-styled-components) extension provides syntax highlighting inside your Javascript files. You can install it as usual from the [Marketplace](https://marketplace.visualstudio.com/items?itemName=jpoissonnier.vscode-styled-components).
 
+### VIM / NeoVim
+The [vim-styled-components](https://github.com/fleischie/vim-styled-components) plugin gives you syntax highlighting inside your Javascript files. Install it with your usual plugin manager like [Plug](https://github.com/junegunn/vim-plug), [Vundle](https://github.com/VundleVim/Vundle.vim), [Pathogen](https://github.com/tpope/vim-pathogen), etc.
+
+Also if you're looking for an awesome javascript syntax package you can never go wrong with [YAJS.vim](https://github.com/othree/yajs.vim) it has all the support you need.
```

Comment:
This "it has all the support you need" makes it seem as if it also has `styled-components` support, which it doesn't. Can we get rid of that sentence and just highlight `vim-styled-components`? Then this is good to go!